Vagrantfile

```diff
@@ -0,0 +1,157 @@
+  # https://discuss.elastic.co/t/es-cluster-in-docker-containers-alreadyclosedexception-underlying-file-changed-by-an-external-force/48874/8
+  if Vagrant.has_plugin?("vagrant-notify-forwarder")
+    config.notify_forwarder.enable = false
+  end
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+
+  config.vm.provision "bootstrapping", type: "shell", privileged: false,
+    keep_color: true, inline: <<-SHELL
+      PURPLE='\033[1;35m'; GREEN='\033[1;32m'; NC='\033[0;0m'
+      echo "${PURPLE}BEGINNING BOOTSTRAPPING PROCESS...Grab a drink. This could take a while.${NC}"
+      echo ' '
+      echo "${PURPLE}UPDATING .BASHRC FILE${NC}"
+      echo "cd /home/ubuntu/#{project_name}" >> /home/ubuntu/.bashrc
+      cd ~/#{project_name}
+      echo "${GREEN}COMPLETED UPDATING .BASHRC FILE${NC}"
+      echo ' '
+      echo "${PURPLE}INSTALLING JAVA${NC}"
+      # Updates apt-get to point to updated java8 package
+      yes | sudo add-apt-repository ppa:webupd8team/java
+      # Updates apt-get
+      sudo apt-get update
+      # Set debconf config file to mark java8 license prompts as completed
+      # https://askubuntu.com/questions/190582/installing-java-automatically-with-silent-option#
+      # Install java8 without prompts
+      echo debconf shared/accepted-oracle-license-v1-1 select true | sudo debconf-set-selections
+      echo debconf shared/accepted-oracle-license-v1-1 seen true | sudo debconf-set-selections
+      yes | sudo apt-get install oracle-java8-installer
+      echo "${GREEN}COMPLETED INSTALLING JAVA${NC}"
+      echo ' '
+      echo "${PURPLE}INSTALLING RVM & RUBY${NC}"
+      yes | sudo apt-get install gnupg2
+      gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+      # Current issue with rvm: https://github.com/rvm/rvm/issues/4068
+      # use this patch until fixed
+      curl -sSL https://raw.githubusercontent.com/wayneeseguin/rvm/stable/binscripts/rvm-installer | bash -s stable --ruby=2.4
+      # uncomment and use below after fixed
+      # curl -sSL https://get.rvm.io | bash -s stable --ruby=2.4
+      source /home/ubuntu/.rvm/scripts/rvm
+      echo "${GREEN}COMPLETED INSTALLING RVM & RUBY${NC}"
+      echo ' '
+      echo "${PURPLE}INSTALLING GEMS${NC}"
+      gem install bundler --no-rdoc --no-ri
+      # also install libcurl3, libcurl3-gnutls, & libcurl4-openssl-dev
+      # which are needed by the "curb" gem, used in some 3rd party rake tasks--tho not actually sure about neo4j
+      yes | sudo apt-get install libcurl3 libcurl4-openssl-dev
+      bundle install
+      gem install overcommit --no-rdoc --no-ri
+      overcommit --install
+      echo "${GREEN}COMPLETED INSTALLING GEMS${NC}"
+  SHELL
+
+  unless Dir["./db/neo4j/development"].any?
```

Comment:
If the project folder doesn't already have neo4j installed `db/neo4j/development`, run the neo4j installation scripts.